tools/ArduinoCsCompiler/ArduinoCsCompiler.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I think for now we should remove this project (as well as all  of the projects underneath this like the samples, Frontend, and tests) from building when build.cmd is invoked using some Exclusions in build.proj file in the root of the repo. 99.9% of the people building the repo won't really care about building them, and this is currently being built since we have:

https://github.com/dotnet/iot/blob/6be64b7477d86aca49503498e862d7be9962489d/build.proj#L38

Alternatively we can disable BuildTools target entirely on the default build.cmd and only enable it when people care about it, so perhaps in your Readme you can add a line saying that they should call build like `build.cmd /p:ToolsBuild=true`

Reply:
I basically agree, but is it worth the extra complexity? The tools take like 2 seconds of the overall build time.